ts/store/helpers/indexer.ts

```diff
@@ -28,10 +30,15 @@ export interface IndexedById<V extends WithId> {
  * @param lst   input list to be indexed (e.g. [ { id: 1, payload: "X" }, { id: 42, payload: "Y" } ] )
  * @returns     indexed object (e.g. { 1: { id: 1, payload: "X" }, 42: { id: 42, payload: "Y" } } )
  */
-export function toIndexed<T extends WithId>(
+export const toIndexed = <T extends WithId>(
   lst: ReadonlyArray<T>
-): IndexedById<T> {
-  return lst.reduce((o, obj) => ({ ...o, [obj.id]: obj }), {} as IndexedById<
-    T
-  >);
-}
+): IndexedById<T> =>
+  lst.reduce((o, obj) => ({ ...o, [obj.id]: obj }), {} as IndexedById<T>);
```

Comment:
use [es6 reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)

Reply:
Uhm isn't it what is already being used here?

Reply:
yes sorry, my fault